teams/components.yml

```diff
@@ -0,0 +1,40 @@
+name: components
+
+purpose: |
+  A team for contributors assisting with triaging resource and reusable
+  task repositories.
+
+members:
+  - lrstanley
+
+responsibilities:
+  - https://github.com/concourse/concourse/blob/master/CODE_OF_CONDUCT.md
```

Comment:
Can we add something here about the team being responsible for looking at issues and PR's that get submitted? That's somewhat conveyed in the `purpose` section, but would be nice to see it clearly stated in this section

Reply:
I added two more, however it would be nice to be able to link to a "how to help us triage issues" style document. I looked through some of the repositories, and I wasn't able to find anything specific to triaging.

If we don't have one, does it make sense to create one? Something that mentions the types of labels that should be used, checking for duplicate issues, reproducing/validating bugs, etc. May be good to have that either in, or at least referenced in, the contributing doc.

Reply:
We don't have anything like that. Maybe add it as a section to the CONTRIBUTING.md in the main concourse/concourse repo?

Reply:
Something similar to these:

- https://github.com/moby/moby/blob/master/project/ISSUE-TRIAGE.md
- https://github.com/kubeflow/community/blob/master/proposals/issue_triage.md
- https://github.com/microsoft/vscode/wiki/Issues-Triaging

Also, as an aside, it might be good to look at potentially using GitHubs new "issue forms", it's a great evolution to issue templates. It is still in beta, however I'm seeing a lot of projects start to use them. I started using them here, as an example:

https://github.com/lrstanley/vault-unseal/issues/new?assignees=&labels=bug&template=bug_report.yml&title=bug%3A+%5BREPLACE+ME%5D

Reply:
Created an issue to followup/track some of the comments above: https://github.com/concourse/concourse/issues/8266